### PR TITLE
Add docsite version number from the library version

### LINKED ISSUE #461: Sphinx should get the sidebar version from the package
The current docsite sidebar pulls the version from a variable hardcoded in conf.py. Instead, it should infer it from the repo. [This SO thread](https://stackoverflow.com/questions/26141851/let-sphinx-use-version-from-setup-py) has some good ideas.

### PR DESCRIPTION
**Pull request by ebezzi**
Fixes #461 

Note that this will only work for tagged releases, otherwise it will add a development version number.

### CODE REVIEW

File: api/python/notebooks/api_demo/census_gget_demo.ipynb
Comment (atolopko-czi) [inline api/python/notebooks/api_demo/census_gget_demo.ipynb:1]: Is this file intended for the PR?

Comment (ebezzi) [reply api/python/notebooks/api_demo/census_gget_demo.ipynb:1]: No, thanks for the catch. Removing this.

**Review by atolopko-czi — APPROVED** [review]
LGTM
